### PR TITLE
Change client ConVars

### PR DESCRIPTION
**Pull request by SquidmarcHD**
why is it fucking failing

### CODE REVIEW

File: Osiris/GUI.cpp [hunk 1]
```diff
@@ -1365,7 +1366,16 @@ void GUI::renderMiscWindow(bool contentOnly) noexcept
         ImGui::EndPopup();
     }
     ImGui::PopID();
-
+    std::string cvar;
```
Comment (notgoodusename) [inline Osiris/GUI.cpp:1369]: make a static string

Comment (SquidmarcHD) [reply Osiris/GUI.cpp:1369]: then itwould only change the first command entered

Comment (g1gantino) [reply Osiris/GUI.cpp:1369]: nice men1! i voted for you !!! hope yo u win election!!

Comment (Cyk-Fad) [reply Osiris/GUI.cpp:1369]: and when it has set, reset this string value

**Comment by techzin** [comment]
the convart set value need a int or bool value, not string.

**Pull request closed without merging**